tools/build/build.js

```diff
@@ -238,6 +238,40 @@ export const DmTarget = new Juke.Target({
   },
 });
 
+export const JustDmTarget = new Juke.Target({
+  parameters: [DefineParameter, DmVersionParameter, WarningParameter, NoWarningParameter],
+  dependsOn: ({ get }) => [
+    get(DefineParameter).includes('ALL_MAPS') && DmMapsIncludeTarget,
+  ],
+  inputs: [
+    '_maps/map_files/generic/**',
+    'maps/**/*.dm',
+    'code/**',
+    'html/**',
+    'icons/**',
+    'interface/**',
```

Comment:
this needs to be rebased to add teh sound folde, I will do that today some point